src/test/scala/com/mwz/sonar/scala/scapegoat/ScapegoatRulesRepositorySpec.scala

```diff
@@ -0,0 +1,55 @@
+package com.mwz.sonar.scala.scapegoat
+
+import inspections.{Level, ScapegoatInspection}
+
+import org.scalatest.{FlatSpec, LoneElement, Matchers}
+import org.sonar.api.server.rule.RulesDefinition.Context
+import org.sonar.api.rule.RuleStatus
+import org.sonar.api.rule.Severity
+import org.sonar.api.rules.RuleType
+
+/** Tests the correct behavior of the Scapegoat Rules Repository */
+class ScapegoatRulesRepositorySpec extends FlatSpec with LoneElement with Matchers {
+  behavior of "the Scapegoat Rules Repository"
+
+  it should "define only one repository" in {
+    val context = new Context()
+    new ScapegoatRulesRepository().define(context)
+
+    context.repositories should have length 1
+  }
+
+  it should "properly define the properties of the repository" in {
+    val context = new Context()
+    new ScapegoatRulesRepository().define(context)
+    val scapegoatRepository = context.repositories.loneElement
+
+    scapegoatRepository.key shouldBe "ScapegoatRepository"
+    scapegoatRepository.name shouldBe "Scapegoat Repository"
+    scapegoatRepository.language shouldBe "scala"
+  }
+
+  it should "define one rule for each scapegoat inspection" in {
+    val context = new Context()
+    new ScapegoatRulesRepository().define(context)
+    val scapegoatRepository = context.repositories.loneElement
+
+    val totalInspections = ScapegoatInspection.AllScapegoatInspections.length
+    scapegoatRepository.rules should have length totalInspections
```

Comment:
I don't know if you would prefer to check against a constant value _(127 in this moment)_.